beta/project-griffon/using-project-griffon/on-device-views.md

```diff
@@ -1,7 +1,7 @@
 # On-device views
 
-## Connected
-
+## Connected 
+(indicated by red Adobe "A")
```

Comment:
add an empty line after the header line. and, remove a space after the `Connected`

Reply:
fixed